GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -1075,8 +1075,12 @@ ProcessingResult QtQuickAdapter::processTXDetails(bs::message::SeqId msgId
          }
       }
       else {
-         itTxDet->second->setDetails(txDet); // shouldn't be more than one entry
-         itTxDet->second->setCurBlock(blockNum_);
+         const auto details = itTxDet->second;
+         QMetaObject::invokeMethod(this, [this, details, txDet] {
```

Comment:
you can use it like `[details=itTxDet->second]` in the capture list (in other PRs)